src_c/rect_impl.h

```diff
@@ -894,6 +900,41 @@ RectExport_moveIp(RectObject *self, PyObject *const *args, Py_ssize_t nargs)
     Py_RETURN_NONE;
 }
 
+static PyObject *
+RectExport_moveTo(RectObject *self, PyObject *const *args, Py_ssize_t nargs,
+                  PyObject *kwnames)
+{
+    PyObject *rect = RectExport_copy(self, NULL);
+    if (!rect) {
+        return NULL;
+    }
+
+    if (nargs > 0) {
+        Py_DECREF(rect);
+        return RAISE(PyExc_TypeError, "move_to takes no positional arguments");
+    }
+
+    if (!kwnames) {
+        Py_DECREF(rect);
+        return RAISE(PyExc_TypeError,
+                     "move_to cannot be called without arguments");
+    }
+
+    Py_ssize_t i, sequence_len;
+    PyObject **sequence_items;
+    sequence_items = PySequence_Fast_ITEMS(kwnames);
+    sequence_len = PySequence_Fast_GET_SIZE(kwnames);
```

Comment:
This can just be PyTuple_GET_SIZE
```suggestion
    sequence_len = PyTuple_GET_SIZE(kwnames);
```